sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2/async_samples/sample_analyze_business_cards_async.py

```diff
@@ -165,19 +165,17 @@ async def main():
     try:
         asyncio.run(main())
     except HttpResponseError as error:
+        print("For more information about troubleshooting errors, see the following guide: "
+              "https://aka.ms/azsdk/python/formrecognizer/troubleshooting")
         # Examples of how to check an HttpResponseError
         # Check by error code:
         if error.error is not None:
-            if error.error.code == "InvalidRequest":
-                print(f"Received an invalid request error: {error.error}")
-                sys.exit(1)
             if error.error.code == "InvalidImage":
                 print(f"Received an invalid image error: {error.error}")
                 sys.exit(1)
         # If the inner error is None and then it is possible to check the message to get more information:
-        filter_msg = ["Generic error", "Timeout", "Invalid request", "InvalidImage"]
-        if any(example_error.casefold() in error.message.casefold() for example_error in filter_msg):
+        if "Invalid request".casefold() in error.message.casefold():
             print(f"Uh-oh! Something unexpected happened: {error}")
             sys.exit(1)
         # Print the full error content:
-        print(f"Full HttpResponseError: {error}")
+        print(f"Received a different error: {error}")
```

Comment:
I wonder if we can just wrap all these lines into the "Received a different error" group to simplify further. Seems the outcome is to print the {error} in either case.

Reply:
I was tempted to keep the check on line 177 to showcase that sometimes errors come in a different format. I could print the message specifically on line 178 to differentiate a bit more, thoughts?

Reply:
Ah I see. it's unfortunate there are two ways to parse the error, but I guess that's just how it is. just fyi, if it hits the "different error" it won't raise an exception /sys.exit. not sure if that was intentional.

Reply:
I did check and I saw it raising the error. I was relying on the programs sys.exit after the try except finished, but I'll add it for symmetry with the other checks.

Reply:
Nevermind, you're right it's not raising the error. I'll add that in

Reply:
Added it, thanks for pointing that out